oneview_redfish_toolkit/util.py

```diff
@@ -436,3 +441,43 @@ def dispatch_event(event):
             globals()['delivery_retry_interval'])
 
         dispatcher.start()
+
+
+def check_oneview_availability(ov_config):
+    """Check OneView availability by doing a GET request to OneView"""
+    attempts = 3
+
+    for attempt_counter in range(attempts):
+        try:
+            connection = HTTPSConnection(
+                ov_config['ip'], context=ssl._create_unverified_context())
+
+            connection.request(
+                method='GET', url='/controller-state.json',
+                headers={'Content-Type': 'application/json'})
+
+            response = connection.getresponse()
+
+            if response.status != 200:
```

Comment:
The Flask API has a file with some status code constants (`from flask_api import status`).